pyproject.toml

```diff
@@ -56,11 +56,18 @@ coverage = ["mqt.bench[test]", "pytest-cov[toml]"]
 docs = [
     "furo>=2023.08.17",
     "sphinx~=7.0",
-    "myst_parser>=0.13",
     "setuptools-scm>=7",
+    "sphinxcontrib-bibtex>=2.4.2",
     "sphinx-copybutton",
-    "sphinx_design",
+    "sphinx-hoverxref",
+    "pybtex>=0.24",
+    "ipython",
+    "ipykernel",
+    "nbsphinx",
     "sphinxext-opengraph",
+    "sphinx-autodoc-typehints",
+    "matplotlib",
+    "pylatexenc",
```

Comment:
If I am not mistaken it should be enough to substitute these for `qiskit[visualization]`. Then you can be sure that everything for printing Qiskit circuits is available.

Reply:
I substituted the requirements as suggested.